views/includes/page-templates/trust-details.html

```diff
@@ -3,7 +3,27 @@
 
     {% include "includes/list/errors.html" %}
 
+    {# Check if there are any beneficial owners. If the length of the 'beneficialOwners' array is 0, then 'isTrustToBeCeased' will be true #}
+    {# and can be used to show the ceased date - only for 'review' mode in update journey #}
```

Comment:
I can see the check for 'review mode' on line 10 but can't see check for 'update journey'.

Reply:
It is implied I think, as the review mode is only available in the update journey. Perhaps i can remove the mention of the update journey in the comment?

Reply:
updated the comment to mention that review mode is part of update journey